vision-zero

```diff
@@ -360,10 +396,12 @@ def get_main_parser():
             "psql",
             "tools-shell",
             "replicate-db",
+            "dump-local-db",
             "remove-snapshots",
             "hasura-console"
         ],
     )
+    parser.add_argument("-f", "--filename", required=False)
```

Comment:
At first, I was going to suggest something more specific that describes what the file needs to be, but then I realized that you can only use the `-f` flag meaningfully after the `replicate-db` command, so 👍.